packages/babel-preset-babili/src/index.js

```diff
@@ -1,20 +1,53 @@
+const err = new Error("Babili is supported only for babel version >=6.14.0");
```

Comment:
We can be more specific here - babel is `babel-core`. Can explain you need to update babel-core, or cli, etc